cfme/storage/volume.py

```diff
@@ -121,22 +167,91 @@ def is_displayed(self):
     form = View.nested(VolumeAddForm)
 
 
+class VolumeCollection(Navigatable):
+    """Collection object for the :py:class:'cfme.storage.volume.Volume'. """
+    def __init__(self, appliance=None):
```

Comment:
**REQUIRED**
Sorry, my bad, `__init__` is not needed here, because it's completely the same as `Navigatable.__init__`.